app/scripts/views/sign_up.js

```diff
@@ -278,6 +284,22 @@ define(function (require, exports, module) {
       return /@firefox(\.com)?$/.test(email);
     },
 
+    _passwordConfirmExperiment: undefined,
+    _isPasswordConfirmEnabled () {
+      if (this._passwordConfirmExperiment === undefined) {
+        this._passwordConfirmExperiment = !! this.isInExperimentGroup('signupPasswordConfirm', 'treatment');
+      }
+      return this._passwordConfirmExperiment;
```

Comment:
This is called in `validEnd`, and `showValidationErrors`, possibly multiple times if the user makes multiple mistakes. There's quite a few branchs, function calls, and especially a hash to determine this cacheable value...